enricher/epss/epss.go

```diff
@@ -0,0 +1,391 @@
+package epss
+
+import (
+	"compress/gzip"
+	"context"
+	"encoding/csv"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"path"
+	"slices"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/quay/zlog"
+
+	"github.com/quay/claircore"
+	"github.com/quay/claircore/enricher"
+	"github.com/quay/claircore/internal/httputil"
+	"github.com/quay/claircore/libvuln/driver"
+	"github.com/quay/claircore/pkg/tmp"
+)
+
+var (
+	_ driver.Enricher          = (*Enricher)(nil)
+	_ driver.EnrichmentUpdater = (*Enricher)(nil)
+)
+
+// EPSSItem represents a single entry in the EPSS feed, containing information
+// about a CVE's Exploit Prediction Scoring System (EPSS) score and percentile.
+type EPSSItem struct {
+	ModelVersion string  `json:"modelVersion"`
+	Date         string  `json:"date"`
+	CVE          string  `json:"cve"`
+	EPSS         float64 `json:"epss"`
+	Percentile   float64 `json:"percentile"`
+}
+
+const (
+	// Type is the type of data returned from the Enricher's Enrich method.
+	Type = `message/vnd.clair.map.vulnerability; enricher=clair.epss schema=https://csrc.nist.gov/schema/nvd/baseURL/1.1/cvss-v3.x.json`
```

Comment:
What was decided for the schema? Should we just go schema-less?

Reply:
Yeah, I agree with schema-less. Is that ok with you guys? @crozzy

Reply:
sure